### DISCUSSION TITLE
GitHubディスカッション体験用

### DISCUSSION THREAD

**Discussion by Ikuyadeu** [vscodejp/handson-hello-vscode-extension#1] [Announcements]
## 👋 ようこそ！
ここでGitHub Discussionsの使い勝手をテストしています．

**Comment by 74th** [comment]
補完の機能は、どのキーボードのキーを押すと動かせるのでしょうか？（Windowsの場合）

**Reply by Ikuyadeu** [reply]
今回の場合，例えば`VS Code`を補完するなら`V`と入力した時点で補完リストが並びます．
補完を手動で呼び出すには<key>Ctrl</key>+<key>Space</key>で呼び出せます．

**Comment by mochan-tk** [comment]
[テストです] ハンズオンの資料はどこにありますか？

**Reply by Ikuyadeu** [reply]
こちらです．
https://github.com/vscodejp/handson-hello-vscode-extension

**Comment by mochan-tk** [comment]
[テストです] ○○というエラーがでました、どうすれば良いでしょうか？
![image](https://user-images.githubusercontent.com/9846618/141644571-ae16aac6-740d-4503-a88a-098ae5a2e968.png)

**Comment by Ikuyadeu** [comment]
[テスト]質問があります．

**Comment by ayatokura** [comment]
expert/01_hello.md の [ビルド](https://github.com/vscodejp/handson-hello-vscode-extension/blob/main/docs/expert/01_hello.md#%E3%83%93%E3%83%AB%E3%83%89) 4:の操作でファイルを開いてみましょう、となっていますが、`.txt`や`.md`ファイルがすぐに開けない場合は、その場で適当にtext.txtを作成することが書いてあると良いかもしれないと思いました。
